tests/spawn-utils.c

```diff
@@ -69,6 +69,19 @@ cmd_result_ensure_ok (CmdResult *cmd,
 }
 
 gboolean
+cmd_result_ensure_ok_verbose (CmdResult *cmd)
+{
+  g_autoptr(GError) error = NULL;
+
+  if (cmd_result_ensure_ok (cmd, &error))
+    return TRUE;
+
+  g_test_message ("%s failure:\n%s", cmd->cmdline, error->message);
```

Comment:
Any reason we don’t just always make this `g_test_message()` call in `cmd_result_ensure_ok()`?